modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - The pattern replacement removes spaces if they were contained in a tag that only contains a space and the text to replace. #TINY-9744
 
+### Fixed
+- `backspaceDeleteIntoListCaret` try to use a pointer to a removed element. #TINY-6888
```

Comment:
This is an internal function describe the error instead that it threw an exception when trying to delete the last BR element in a list element or similar. Think of the reader here they won't know or care where the fix was made more what the fix was for.

- [ ] Update changelog to reflect the change from the user perspective

Reply:
@spocke done, let me know if it is ok